config/schedule.rb

```diff
@@ -21,9 +21,25 @@
 set :job_template, "/bin/sh -l -c ':job'"
 set :environment, ENV["RAILS_ENV"]
 
-every 1.day do
-  runner "Importer.new(:poi)"
-  runner "Importer.new(:event)"
+every 1.day, at: "00:05 am" do
+  runner "Importer.load_data(:poi)"
+end
+
+every 1.day, at: "00:30 am" do
+  runner "Importer.load_data(:event)"
+end
+
+every 1.day, at: "01:05 am" do
+  runner "Importer.parse_data(:poi)"
+end
+
+every 1.day, at: "03:05 am" do
+  runner "Importer.parse_data(:event)"
+end
+
+every 1.day, at: "05:05 am" do
+  runner "Importer.send_data(:poi)"
+  runner "Importer.send_data(:event)"
```

Comment:
wieso sind beim `parse_data` zwei stunden zwischen `poi` und `event` und beim `send_data` nicht, sondern zeitgleich?

Reply:
Das Parsen der Daten wird aus dem XML per Nokogiri geladen (250MB Analyse). Das dauert unter Umständen etwas.
Das Senden der Datensätze erfolgt dann aus der Datenbank seriell ein Eintrag nach dem anderen. Das sollte meiner Meinung nach flott gehen. die beiden Exporte werden zwar zeitgleich gestartet aber in einem eigenen Job unabhängig voneinander.